### PR TITLE
Upgrade ruby version, possibly closing  #403

### PR DESCRIPTION
**Pull request by lislis**

### CODE REVIEW

**Comment by til** [comment]
:+1:

**Comment by PragTob** [comment]
:+1:

**Comment by lislis** [comment]
This should be fine now. If someone wants to hit the big green button :)

File: db/schema.rb [hunk 1]
```diff
@@ -65,6 +65,14 @@
...
   add_index "memberships", ["person_id"], name: "index_memberships_on_person_id"
 
+  create_table "notifications", force: true do |t|
+    t.integer  "person_id"
+    t.datetime "viewed_at"
+    t.string   "content"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
```
Comment (til) [inline db/schema.rb]: Meep, unrelated change, meep.

Comment (til) [inline db/schema.rb]: How to avoid: one way is to more often run `bin/rake db:reset` locally. This way, when working on a different branch, there is less risk that your database has unwanted changes from another branch.

Comment (lislis) [inline db/schema.rb]: So, how do I get this out of there now?

Comment (til) [inline db/schema.rb]: Add a new commit to this branch that removes it. Either push it as new
commit, or, even better, to get a clean history for the master branch,
squash and force push (this branch, not master).

**Comment by lislis** [comment]
@til removed it!

**Comment by til** [comment]
Cool, thanks! :rocket:

**Comment by lislis** [comment]
Lady Gaga
